src/ontology/edits/oeo-physical.omn

```diff
@@ -3733,6 +3737,248 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/768",
         not (<http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010090)
     
     
+Class: OEO_00010095
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Marine thermal energy is a kind of natural ambient thermal energy that is present in marine water bodies.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine thermal energy"@en
+    
+    SubClassOf: 
+        OEO_00140104
+    
+    
+Class: OEO_00010096
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Marine thermal energy transfer is a heat transfer from the marine water body to a transportable material entity.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/762
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/806",
+        rdfs:label "marine thermal energy transfer"@en
+    
+    SubClassOf: 
+        OEO_00140101,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00010105
+    
+    
+Class: OEO_00010097
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Marine current energy is the hydro energy of moving marine water.",
```

Comment:
I suggested a change to this definition later on in the discussion which probably got lost: _Marine current energy is the hydro energy of **an ocean current.**_